main/webapp/modules/core/scripts/views/data-table/text-transform-dialog.html

```diff
@@ -1,4 +1,4 @@
-<div class="dialog-frame" >
+<div class="dialog-frame" style="width: 80%;" >
```

Comment:
Wouldn't it be cleaner to store that in CSS? It's generally recommended to avoid inline styling.

Reply:
The approach @jackyq2015 has taken here is consistent with all the other 'dialog-frame' divs in the app at the moment. IMO ideally all these should be done instead by adding a class to the div and styling via css. BUT, I'd suggest consistency is better than doing this one instance 'correctly' - then we can tidy them all together (I'd be happy to take this on once this PR is merged)

Reply:
agree

Reply:
Team, we know its best practice, but let's not focus too much on the frontend incidentals.  Our plan is for a UI overhaul with Google's help.  Honestly, we just need to work on the backend and just "fast quick repair" on the frontend to make things "work good enough for now", knowing that our UI re-work can happen with Google's assistance in a bit.  I already talked to Jacky about this during my hang out with him.  Simon@Google is working on getting a designer or two for helping us out.

Speaking of hangouts, we need to come to some regularly scheduled hangouts for the 4 of us.  Oh @ostephens if you didn't know, we kinda voted you as a core contributor in the past day :) and I'll send out an official email shortly, announcing.